### PR TITLE
Created folder structure, added the components and pages 

### PR DESCRIPTION
**Pull request by Vigneshsubr**
Created dummy components are Button, Label and Input then pages are SignUp, Login , Update and ListOfData

### CODE REVIEW

**Comment by Santhosh-BE** [comment]
@Vigneshsubr  

1. create a proper components file 
2. folder structure need to beproper

File: react-api-integration/src/components/Buttton.jsx
Comment (Santhosh-BE) [inline react-api-integration/src/components/Buttton.jsx:1]: Do not create empty files for components. Ensure each component file has a proper structure and content.

File: react-api-integration/src/components/Input.jsx
Comment (Santhosh-BE) [inline react-api-integration/src/components/Input.jsx:1]: Do not create empty files for components. Ensure each component file has a proper structure and content.

File: react-api-integration/src/components/Label.jsx
Comment (Santhosh-BE) [inline react-api-integration/src/components/Label.jsx:1]: Do not create empty files for components. Ensure each component file has a proper structure and content.

**Review by Santhosh-BE — CHANGES_REQUESTED** [review]

File: react-api-integration/src/components/Buttton.jsx
Comment (Vigneshsubr) [reply react-api-integration/src/components/Buttton.jsx:1]: Now created the Folders as assets, utils  and redux folders

File: react-api-integration/src/components/Input.jsx
Comment (Vigneshsubr) [reply react-api-integration/src/components/Input.jsx:1]: now solved anna

File: react-api-integration/src/components/Label.jsx
Comment (Vigneshsubr) [reply react-api-integration/src/components/Label.jsx:1]: now solved

**Pull request merged**